README.md

```diff
@@ -10,65 +11,77 @@ Running the full ISAR system requires an installation of a robot which satisfies
 ### Give Docker containers access to the Nvidia graphics card on the host machine
 
 #### Pre-requisites
+
 - git
 - docker
 - docker-compose
 
-NOTE: Docker must NOT be installed using Snap. The Snap version is not compatible with nvidia-docker2. Instead, follow the [official documentation](https://docs.docker.com/engine/install/ubuntu/) from Docker for installation. 
+NOTE: Docker must NOT be installed using Snap. The Snap version is not compatible with nvidia-docker2. Instead, follow the [official documentation](https://docs.docker.com/engine/install/ubuntu/) from Docker for installation.
 
 - To check if your Docker version was installed using snap run the following command.
-	```sh
-	systemctl list-units --type=service | grep docker
-	```
-	If the result is `snap.docker.dockerd.service` then the installation has been done using snap and must be reinstalled.
+  ```sh
+  systemctl list-units --type=service | grep docker
+  ```
+  If the result is `snap.docker.dockerd.service` then the installation has been done using snap and must be reinstalled.
 
 #### Installation nvidia docker
+
 1. Install the newest recommended drivers from Nvidia if not already as described by the following [documentation](https://linuxconfig.org/how-to-install-the-nvidia-drivers-on-ubuntu-20-04-focal-fossa-linux)
 2. Install nvidia-docker2 using the [official documentation](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html)
 
 ### Run simulation
 
 Build the container. This needs to be done once before one can give the container access to the screen.
+
 ```bash
 $ docker-compose build
 ```
 
 Give the docker container access to the screen, this needs to be done each time the computer is restarted.
+
 ```bash
 $ xhost +local:`docker inspect --format='{{ .Config.Hostname }}' turtle_sim`
 ```
 
 Start the simulation
+
 ```bash
 $ docker-compose up
 ```
 
-Build and start the simulation
+To build and start the simulation
+
 ```bash
 $ docker-compose up --build
 ```
 
+The simulation world that is used can be set by changing the world variable in the 'entrypoint.sh' file.
+
+## Adding new models
+
+New models can be added by placing the model that is used by Gazebo into '/models/new_world/' and adding a "new_world.world" file into 'ros_packages/isar_turtlebot/worlds/'. The map that is used by the planner should be placed into '/ros_packages/isar_turtlebot/maps/' with the name 'new_world.\*'. To add a default configuration for the initial pose and position of the robot in the simulation, add '/config/new_world.cfg' with the desired parameters.
```

Comment:
Should we add a sentence about changing the "WORLD_NAME" parameter in the docker file? if you want to run on an other world than "house"